### DISCUSSION TITLE
How can I get start with Microsoft student ambassador program?

### DISCUSSION THREAD

**Discussion by SusheelThapa** [Schweinepriester/github-profile-achievements#652] [Q&A]
How can I get start with Microsoft student ambassador program?

**Comment by Nirzak** [comment]
You have to apply from here https://mvp.microsoft.com/studentambassadors but remember The minimum age requirement for applicants is 16 years. Full-time enrollment in an accredited academic institution (such as a College or University) is required for applicants.

**Comment by ArjavJain-27** [comment]
If you're starting with the Microsoft Student Ambassadors program, the process is a little different now.

You don't need to wait for a traditional application or interview. Microsoft has redesigned the program around learning, community contribution, and practical activities.

A good way to start:

1. Go to the Microsoft Student Ambassadors page and click **Get Started**.
2. Register and verify your student status. You may need to link an **Azure for Students** account.
3. Complete the activities Microsoft gives you based on your interests and skills.
4. Focus on actually contributing — for example, build AI projects, share useful technical content, organize student events, or help other students learn.
5. Keep an eye on your registration email. Students who meet the activity requirements can receive an invitation to join as an official Student Ambassador.

You don't need to be an expert before starting. I'd recommend picking one area you're genuinely interested in (AI, Azure, GitHub, etc.) and consistently building and sharing things around it.

Official info: Microsoft Student Ambassadors → https://mvp.microsoft.com/studentambassadors

Hope this helps!
